pkg/trivyk8s/trivyk8s.go

```diff
@@ -105,6 +110,35 @@ func (c *client) ListArtifacts(ctx context.Context) ([]*artifacts.Artifact, erro
 	return artifactList, nil
 }
 
+// ListArtifacts returns kubernetes scannable artifacs.
+func (c *client) ListArtifactAndNodeInfo(ctx context.Context) ([]*artifacts.Artifact, error) {
+	artifactList, err := c.ListArtifacts(ctx)
+	if err != nil {
+		return nil, err
+	}
+	jc := jobs.NewCollector(c.cluster, jobs.WithTimetout(time.Minute*5), jobs.WithJobTemplateName(jobs.NodeCollectorName))
+	// delete trivy namespace
+	defer jc.Cleanup(ctx)
```

Comment:
We cannot clean the namespace if it existed before, or we need a new namespace. Running the example here to get the nodes info deleted my trivy-operator `trivy-system` namespace!